index.html

```diff
@@ -8,41 +9,343 @@
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" id="fourth-item">
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+
+    </div>
+
+    <div class="about-me">
+
+        <div id="title">
+            <h2>About me</h2>
+        </div>
+        <div class="about-me-body">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </div>
+        <div id="btn-2">
+            <button type="button" class="btn-2">Get My Resume</button>
```

Comment:
I see that you forgot to implement the button style on hover and active, I suggest implementing that 
![image](https://user-images.githubusercontent.com/61790587/164762243-f5ab6e78-7f45-4369-aabf-93d16320b044.png)